org.tech.JTDX.yml

```
@@ -0,0 +1,66 @@
+app-id: org.tech.JTDX
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: jtdx
+rename-icon: jtdx
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+
+modules:
+  - shared-modules/libusb/libusb.json
+  - shared-modules/linux-audio/fftw3f.json
+
+  - name: portaudio
+    sources:
+      - type: archive
+        url: http://files.portaudio.com/archives/pa_stable_v190600_20161030.tgz
+        sha256: f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+
+  - name: boost
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.81.0/source/boost_1_81_0.tar.bz2
+        sha256: 71feeed900fbccca04a3b4f2f84a7c217186f28a940ed8b7ed4725986baf99fa
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=$FLATPAK_DEST --with-libraries=container,date_time,filesystem,log
+      - ./b2 -j $FLATPAK_BUILDER_N_JOBS install
+
+  - name: hamlib
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/4.5.4/hamlib-4.5.4.tar.gz
+        sha256: b1aea97d6093990b77d5dc4bde6b9ca06183ddb7e24da7e2367a2acc957b7ac2
+
+  - name: jtdx
+    sources:
+      - type: archive
+        url: https://github.com/jtdx-project/jtdx/archive/159/jtdx-159.tar.gz
+        sha256: 85b02fbe7f1d53e29dd592fc78e32a458a2facf6ae08bbd24ec2b1fe16155dcc
+      - type: file
+        path: org.tech.JTDX.metainfo.xml
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DWSJT_SKIP_MANPAGES=ON
+      - -DWSJT_GENERATE_DOCS=OFF
+    post-install:
+      - install -D org.tech.JTDX.metainfo.xml /app/share/metainfo/org.tech.JTDX.metainfo.xml
+      - mkdir -p /app/share/icons/hicolor/64x64/apps/
+      - cp /app/share/pixmaps/jtdx_icon.png /app/share/icons/hicolor/64x64/apps/jtdx_icon.png
```

```suggestion
      - install -Dm644 /app/share/pixmaps/jtdx_icon.png /app/share/icons/hicolor/64x64/apps/jtdx_icon.png
```

And we require at least 128x128 icon so this might not be enough.

---

ok tks